test/test_results.py

```diff
@@ -0,0 +1,158 @@
+import unittest
+import sys
+import os
+import subprocess
+import warnings
+
+# Add the parent directory to the Python path
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+from test.utils.custom_tsv_parser import ParseTSVFile
+
+PREF_CUTOFF = -4
```

Comment:
Why is this hardcoded here? 🤔 , If there is a setting for it this should be mocked or patched depending on the need.